src/NavigationActions.js

```diff
@@ -9,6 +9,8 @@ const INIT = 'Navigation/INIT';
 const NAVIGATE = 'Navigation/NAVIGATE';
 const RESET = 'Navigation/RESET';
 const SET_PARAMS = 'Navigation/SET_PARAMS';
+const SET_TRANSITION_END = 'Navigation/SET_TRANSITION_END';
```

Comment:
Should we add these to the NavigationActions docs https://reactnavigation.org/docs/navigators/navigation-actions?

Reply:
That's a good idea. Coming up with the solution is apparently 25% of submitting a PR. I will also add tests.